.github/workflows/ci.yml

```diff
@@ -13,8 +13,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        otp: ['24', '25']
-        rebar3: ['3.16.1']
+        include:
+          - otp: 27
+            os: ubuntu-24.04
+            rebar3: 3.23
+            dialyzer: true
```

Comment:
Lemme know if this is acceptable. The reason for this exception is that with e.g. OTP 25 Dialyzer fails by not knowing option `no_unknown`.